dtool_lookup_gui/models/datasets.py

```diff
@@ -334,10 +329,12 @@ def put_readme(self, text):
 
     async def get_readme(self):
         if 'readme_content' in self._dataset_info:
+            logger.debug("README.yml cached.")
             return self._dataset_info['readme_content']
-        async with ConfigurationBasedLookupClient() as lookup:
```

Comment:
If we want to support multiple lookup endpoints, we need to keep the OO-type API.

Reply:
That's true, then instead of phasing it out here, I will document this core of the API properly at some point (https://github.com/livMatS/dtool-lookup-api/issues/27).